src/librustc_typeck/check/method/suggest.rs

```diff
@@ -271,11 +271,35 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                     let mut candidates = all_traits(self.tcx).into_iter().filter_map(|info| {
                         self.associated_item(info.def_id, item_name, Namespace::ValueNS)
                     });
-                    if let (true, false, SelfSource::MethodCall(expr), Some(_)) = (
+                    // There are methods that are defined on the fundamental types and won't be
```

Comment:
```suggestion
                    // There are methods that are defined on the primitive types and won't be
```
I was initially confused thinking that this was about `#[fundamental]`